src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -49,6 +50,7 @@ const Sidebar: React.FC = () => {
   return (
     <div className={styles.sidebar}>
       <NavPageLinks linkItems={linkItems} />
+      <Typography className={styles.versionNumber}>v.3.5.1</Typography>
```

Comment:
i dont see this on my screen for some reason (also now it'll be 3.5.3 when this gets released probably)